src/dayview.html

```diff
@@ -1,187 +1,181 @@
-              </table>
-          </init-position-scroll>
-      </div>
+        </div>
+        <init-position-scroll *ngIf="sliderIndex === currentViewIndex"
+          class="dayview-normal-event-container dayview-categorized-normal-event-container"
+          [initPosition]="initScrollPosition" [emitEvent]="preserveScrollPosition"
+          (onScroll)="setScrollPosition($event)">
+          <table class="table table-bordered table-fixed dayview-normal-event-table">
+            <tbody>
+              <tr *ngFor="let row of views[sliderIndex].rows; let i = index">
+                <td class="calendar-hour-column text-center">
+                  {{ hourColumnLabels[i] }}
+                </td>
+                <td *ngIf="dayviewCategorySource?.size === 0" class="calendar-cell" (click)="select(row)"
+                  tappable>
+                  <ng-template [ngTemplateOutlet]="dayviewNormalEventSectionTemplate"
+                    [ngTemplateOutletContext]="{tm: { events: [], time: row.time }, hourParts, eventTemplate: dayviewNormalEventTemplate}">
+                  </ng-template>
+                </td>
+                <td *ngFor="let category of dayviewCategorySource" class="calendar-cell"
+                  (click)="select(row, category)" tappable>
+                  <ng-template *ngIf="row.eventsGroupByCategory?.get(category)?.length"
+                    [ngTemplateOutlet]="dayviewNormalEventSectionTemplate"
+                    [ngTemplateOutletContext]="{tm: { events: row.eventsGroupByCategory?.get(category)!, time: row.time }, hourParts, category, eventTemplate: dayviewNormalEventTemplate}">
+                  </ng-template>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </init-position-scroll>
+
+        <init-position-scroll *ngIf="sliderIndex !== currentViewIndex"
+          class="dayview-normal-event-container dayview-categorized-normal-event-container"
+          [initPosition]="initScrollPosition">
+          <table class="table table-bordered table-fixed dayview-normal-event-table">
+            <tbody>
+              <tr *ngFor="let row of views[sliderIndex].rows; let i = index">
+                <td class="calendar-hour-column text-center">
+                  {{ hourColumnLabels[i] }}
+                </td>
+                <td *ngIf="dayviewCategorySource?.size === 0" class="calendar-cell" (click)="select(row)"
+                  tappable>
+                  <ng-template [ngTemplateOutlet]="dayviewInactiveNormalEventSectionTemplate"
+                    [ngTemplateOutletContext]="{tm: { events: [], time: row.time }, hourParts, eventTemplate: dayviewNormalEventTemplate}">
+                  </ng-template>
+                </td>
+                <td *ngFor="let category of dayviewCategorySource" class="calendar-cell"
+                  (click)="select(row, category)" tappable>
+                  <ng-template *ngIf="row.eventsGroupByCategory?.get(category)?.length"
```

Comment:
I think even current category doesn't have event, you still want to display the style for dayviewInactiveNormalEventSectionTemplate, so should not use *ngIf to exclude it. Otherwise the cells having event and not having event may have different style.